### PR TITLE
Ritual - Elementalist Mastermind of Discord

### PR DESCRIPTION
**Pull request by uilman**
Frost Bomb
image
No Frost Bomb, exposure from Corrosive Elements
image
Fireball with added lightning linked, lightning exposure, no fire exposure
image

Condition for mana regen
image

### CODE REVIEW

**Comment by uilman** [comment]
image

**Comment by Wires77** [comment]
I'd love if we didn't have an extra config option for "Have you applied exposure recently?" and just check to see if the enemy has one of the exposures instead.  If we can't do enemy mods in ModParser, there are config options already for "Is the enemy exposed to ___?"

**Comment by uilman** [comment]
See changes, now applies condition when we apply an exposure effect.

File: Modules/CalcPerform.lua [hunk 1]
```diff
@@ -1629,7 +1629,13 @@ function calcs.perform(env)
 			end
 		end
 		if min ~= math.huge then
+			-- Modify the magnitude of all exposures
+			for i, value in ipairs(modDB:Tabulate("BASE", nil, "ExtraExposure")) do
```
Comment (ppoelzl) [inline Modules/CalcPerform.lua:1633]:
```suggestion
			for _, value in ipairs(modDB:Tabulate("BASE", nil, "ExtraExposure")) do
```

Comment (ppoelzl) [reply Modules/CalcPerform.lua:1633]: Avoid named unused variable.

**Review by ppoelzl — APPROVED** [review]
LGTM.

**Pull request merged**